src/platform/forms-system/src/js/fields/FileField.jsx

```diff
@@ -334,6 +334,22 @@ class FileField extends React.Component {
               const deleteButtonText =
                 enableShortWorkflow && hasErrors ? 'Cancel' : 'Delete file';
 
+              const fileId = `file-name-${index}`;
+
+              const getUiSchema = innerUiSchema =>
+                typeof innerUiSchema === 'function'
+                  ? innerUiSchema({ fileId, index })
+                  : innerUiSchema;
```

Comment:
How does this relate to the labeling changes?

Reply:
The file upload may or may not include an `attachmentSchema` or `attachmentName` in the `uiSchema`. And since the `aria-describedby` needs to be in the attached to whatever form element is included it gets defined in the `widgetProps` value of the `ui:options`.

The `fileId` needs to be passed to the uiSchema (code below copied from the all-claims changes in this PR) which can only be done if the `attachmentSchema` is a function.

```js
attachmentSchema: ({ fileId }) => ({
  'ui:title': 'Document type',
  'ui:disabled': isDisabled,
  'ui:widget': widgetType,
  'ui:options': {
    widgetProps: {
      'aria-describedby': fileId,
    },
  },
}),
```